src/pages/_app.tsx

```diff
@@ -39,10 +40,25 @@ type AppPropsWithLayout = AppProps & {
   Component: NextPageWithLayout;
 };
 
+const hostUrl =
+  process.env.NODE_ENV === 'development'
+    ? 'localhost:3000'
+    : process.env.VERCEL_URL;
```

Comment:
Using VERCEL_URL here makes it so testing with gitpod doesn't quite work.  But this URL does work
https://3000-veganhackti-veganhackti-yb9g5zc3oib.ws-us80.gitpod.io/_next/static/media/VH_Logo_Crest_Tagline_Gray.98a43ba2.png